.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "FV3"]
   path = FV3
-  url = https://github.com/NOAA-EMC/fv3atm
-  branch = develop
+  url = https://github.com/rmontuoro/fv3atm
+  branch = bugfix/thompson-tracer-index
```

Comment:
@rmontuoro Please make an fv3atm PR, we need to merge your bugfix/thompson-tracer-index branch to emc develop branch.

Reply:
See [PR#509](https://github.com/NOAA-EMC/fv3atm/pull/509).